contrib/terraform/exoscale/README.md

```diff
@@ -0,0 +1,117 @@
+# Kubernetes on Exoscale with Terraform
+
+Provision a Kubernetes cluster on Exoscale using Terraform and Kubespray
+
+## Overview
+
+The setup looks like following
+
+```
+                           Kubernetes cluster
+                        +-----------------------+
++---------------+       |   +--------------+    |
+|               |       |   | +--------------+  |
+| API server LB +---------> | |              |  |
+|               |       |   | | Master/etcd  |  |
++---------------+       |   | | node(s)      |  |
+                        |   +-+              |  |
+                        |     +--------------+  |
+                        |           ^           |
+                        |           |           |
+                        |           v           |
++---------------+       |   +--------------+    |
+|               |       |   | +--------------+  |
+|  Ingress LB   +---------> | |              |  |
+|               |       |   | |    Worker    |  |
++---------------+       |   | |    node(s)   |  |
+                        |   +-+              |  |
+                        |     +--------------+  |
+                        +-----------------------+
+```
+
+## Requirements
+
+* Terraform 0.12.0 or newer
+
+## Quickstart
+
+Edit `tfvars.json` to match your setup
```

Comment:
Makes sense, I used this just because it's what we're using and I didn't want to change it. But I have changed so much anyway so I think it totally makes sense

Reply:
:+1: